python-sdk/python/eppo_client/_eppo_client.pyi

```diff
@@ -0,0 +1,171 @@
+        is_graceful_mode: bool = True,
+        poll_interval_seconds: int | None = ...,
+        poll_jitter_seconds: int = ...,
+        initial_configuration: Configuration | None = None
+    ): ...
+
+class AssignmentLogger:
+    def log_assignment(self, event: Dict) -> None: ...
+    def log_bandit_action(self, event: Dict) -> None: ...
+
+class EppoClient:
+    def get_string_assignment(
+        self,
+        flag_key: str,
+        subject_key: str,
+        subject_attributes: Dict[str, Union[str, int, float, bool, None]],
+        default: str,
+    ) -> str: ...
+    def get_integer_assignment(
+        self,
+        flag_key: str,
+        subject_key: str,
+        subject_attributes: Dict[str, Union[str, int, float, bool, None]],
+        default: int,
+    ) -> int: ...
+    def get_numeric_assignment(
+        self,
+        flag_key: str,
+        subject_key: str,
+        subject_attributes: Dict[str, Union[str, int, float, bool, None]],
+        default: float,
+    ) -> float: ...
+    def get_boolean_assignment(
+        self,
+        flag_key: str,
+        subject_key: str,
+        subject_attributes: Dict[str, Union[str, int, float, bool, None]],
+        default: bool,
+    ) -> bool: ...
+    def get_json_assignment(
+        self,
+        flag_key: str,
+        subject_key: str,
+        subject_attributes: Dict[str, Union[str, int, float, bool, None]],
+        default: Any,
+    ) -> Any: ...
+    def get_string_assignment_details(
+        self,
+        flag_key: str,
+        subject_key: str,
+        subject_attributes: Dict[str, Union[str, int, float, bool, None]],
+        default: str,
+    ) -> EvaluationResult: ...
+    def get_integer_assignment_details(
+        self,
+        flag_key: str,
+        subject_key: str,
+        subject_attributes: Dict[str, Union[str, int, float, bool, None]],
+        default: int,
+    ) -> EvaluationResult: ...
+    def get_numeric_assignment_details(
+        self,
+        flag_key: str,
+        subject_key: str,
+        subject_attributes: Dict[str, Union[str, int, float, bool, None]],
+        default: float,
+    ) -> EvaluationResult: ...
+    def get_boolean_assignment_details(
+        self,
+        flag_key: str,
+        subject_key: str,
+        subject_attributes: Dict[str, Union[str, int, float, bool, None]],
+        default: bool,
+    ) -> EvaluationResult: ...
+    def get_json_assignment_details(
+        self,
+        flag_key: str,
+        subject_key: str,
+        subject_attributes: Dict[str, Union[str, int, float, bool, None]],
+        default: Any,
+    ) -> EvaluationResult: ...
+    def get_bandit_action(
+        self,
+        flag_key: str,
+        subject_key: str,
+        subject_context: (
+            ContextAttributes | Dict[str, Union[str, int, float, bool, None]]
+        ),
+        actions: (
+            Dict[str, ContextAttributes]
+            | Dict[str, Dict[str, Union[str, int, float, bool, None]]]
```

Comment:
For later, we should also support contextless actions. I know we try to steer customers away from this, but I've seen this use case come up on many calls.

Reply:
Do you mean supporting `actions: Dict[str, None]` or `list[str]`? We should be able to add this in the future in a backward-compatible way